result/isislsp.yaml

```diff
@@ -0,0 +1,341 @@
+components:
+  schemas:
+    IsisLsps.State.Request:
+      description: >-
+        The request to retrieve BGP peer prefix information.
+      type: object
+      properties:
+        isis_router_names:
+          description: >-
+            The names of ISIS Routers to return results for.
+            An empty list will return results for all ISIS router.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.IsisRouter/properties/name"
+        lsp_filters:
+          description: >-
+            Specify which LSPs to return. If the list is empty or missing
+            then all LSPs will be returned, e.g. '6400000000010000'.
+          type: array
+          items:
+            type: string
+            format: hex
+  
+    IsisLsps.State:
+      description: >-
+        ISIS LSP.
+      type: object
+      properties:
+        isis_router_name:
+          description: >-
+            The name of a ISIS Router.
+          type: string
+        lsps:
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.State'
+
+    IsisLsp.State:
+      description: >-
+        ISIS LSP.
+      type: object
+      properties:
+        lsp_id:
+          description: >-
+            LSP ID of the LSP, e.g. '6400000000010000'.
+          type: string
+          format: hex
+        pdu_type:
+          description: >-
+            Link State PDU type.
+          type: string
+          enum: [level_1, level_2]
+        remaining_lifetime:
+          description: >-
+            Remaining lifetime in seconds before the LSP expiration.
+          type: integer
+          format: int64
+          minimum: 0
+          maximum: 4294967295
+        sequence_number:
+          description: >-
+            Sequence number of the LSP.
+          type: integer
+          format: int64
+          minimum: 0
+          maximum: 4294967295
+        pdu_length:
+          description: >-
+            Total length of the LSP.
+          type: integer
+          minimum: 0
+          maximum: 65535
+        flags:
+          description: >-
+            LSP Type-Block flags.
+          $ref: '#/components/schemas/IsisLsp.Flags'
+        is_type:
+          description: >-
+            Type of neighboring system.
+          type: integer
+          minimum: 1
+          maximum: 3
+        hostname:
+          description: >-
+            This container defines TLV 137.
+          type: string
+        is_reachability:
+          description: |-
+            This container describes list of ISIS neighbors and attributes.
+          $ref: '#/components/schemas/IsisLsp.IsReachability'
+        extended_is_reachability:
```

Comment:
How is this field different than 'is_reachability'?  The descriptions of both are same, doesn't seem like they should be same.